packages/nx/src/utils/async-iterator.ts

```diff
@@ -0,0 +1,16 @@
+export function isAsyncIterator<T>(v: any): v is AsyncIterableIterator<T> {
+  return typeof (v as any)?.[Symbol.asyncIterator] === 'function';
+}
+
+export async function getAsyncIteratorResult<T>(
```

Comment:
This is more like `getLastValueFromAsyncIterator` since it ignores all other values.

Reply:
Agreed. I just moved the function from its original implementation location. I didn't stop to look at it. I'll update it.